_includes/navbar.ext

```diff
@@ -75,7 +75,7 @@
               <ul class="dropdown-menu" aria-labelledby="about_menu">
                 <li><a href="{{ site.baseurl }}/get_involved.html">Get involved!</a><li>
                 <li><a href="{{ site.baseurl }}/organization/team.html">The Coordination Team</a></li>
-                <li><a href="{{ site.baseurl }}/organization/planning/plan-of-work-2021.html">HSF Planning</a></li>
```

Comment:
Out of curiosity - what will then happen with the 2021 planning page? It stays in the site, I see, but then it won't appear anymore anywhere? That would be a shame. We could easily do what we did for the CWP page with a "history" section for previous years, see https://hepsoftwarefoundation.org/organization/cwp.html. Seems appropriate to me.

Reply:
Ah, answering myself - I now see the link for 2021 below :D.

Reply:
Yes, I thought the best is that the planning always points to the current year (so not having to click twice), but with links to previous years always available